guides/Overview.md

```diff
@@ -1,9 +1,19 @@
 # Introduction
 
-Velo's APIs (previously Corvid) empower you to take full control of your site’s functionality. Use the APIs to interact with site elements, your site’s database content, Wix Apps, and external services. The APIs also give you access to information about your site, its users, and more.
+Welcome to Wix's Velo API Reference. This reference is your comprehensive guide to using Velo, Wix's powerful JavaScript-based development platform, to create dynamic and interactive web experiences. Velo empowers you to take full control of your site’s functionality. Use Velo APIs to fully customize your site, from frontend design to backend functionality.
 
-To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. The APIs include lots of code examples to help you [get started](tutorials/getting-started).
+The API reference provides detailed documentation and code examples for all APIs, functions and event handlers, and properties that you can use to interact with various aspects of your site. To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. 
 
+With Velo you can:
+
+
++ **Access Site Elements**: Use the [Wix Editor Elements($w)](https://www.wix.com/velo/reference/$w) API to work with your site's elements and handle user interactions.
+
++ **Manage Database Collections**: Programmatically interact with your site's database collections using the [Data API](https://www.wix.com/velo/reference/wix-data).
+
++ **Create Custom eCommerce Solutions**: The [Wix eCommerce API](https://www.wix.com/velo/reference/wix-ecom-backend) allows you to manage your products, orders, fulfillment, and more.
+
+>**New to Velo?** Check out our [getting started guide](https://dev.wix.com/docs/develop-websites/articles/getting-started/getting-started-with-code) to get you quickly up and running with code.
 
 ## Release Notes
 
```

Comment:
Consider changing this sentence so "release notes" isn't capitalized.:

"Check out our [Release Notes](https://www.wix.com/velo/reference/release-notes/release-notes) to see a list of all the latest changes, updates, and additions to our API Reference."